app/components/layout/keyboard_layout/keyboard_layout.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {KeyboardAvoidingView, Platform, View} from 'react-native';
+
+export default class KeyboardLayout extends Component {
+    static propTypes = {
+        behaviour: PropTypes.string,
+        children: PropTypes.node,
+        keyboardVerticalOffset: PropTypes.number,
+        statusBarHeight: PropTypes.number,
+        watchStatusBar: PropTypes.bool
```

Comment:
What does watchStatusBar do?

Reply:
In iOS when you are in a call or something like that the status bar height changes like in the screenshots and this takes care of that
![image](https://user-images.githubusercontent.com/6757047/31448442-d96fd2a4-ae7a-11e7-9889-892127cbb22c.png)
![image](https://user-images.githubusercontent.com/6757047/31448459-df8e364e-ae7a-11e7-962b-076655bea5f4.png)

Reply:
Oh I see. I didn't see it was actually used anywhere. Shouldn't we just have this view watch the status bar at all times?

Reply:
sure we can do that, I don't see why not